lib/actions/jlinkTargetActions.js

```diff
@@ -93,9 +99,36 @@ const getDeviceInfo = serialNumber =>
                 nrfjprog.getDeviceInfo(
                     serialNumber,
                     (deviceInfoError, devInfo) => {
+                        const serialNumberPrefixNrf52 = '682';
```

Comment:
Consider using capital letters to make it stand more out as a constant. Same with the device info below.

Reply:
Good point!